_data/home.yml

```diff
@@ -9,10 +9,14 @@ navbar_entries:
     url: https://www.abhinav.co
 
 project_entries:
-  - title: Practical 1
-    url: prac1-device-setup
+  - title: Practical 1a, device setup
+    url: prac1a-device-setup
     desc: Data collection
```

Comment:
Instead of naming the new tutorials into 1a and 1b? Does it make more sense if we name them to pratical 1 and 2 instead? It seems a bit strange if only the first two practicals have letter subheadings? But this is me being a bit pedagogical.

![Screenshot 2022-11-25 at 16 16 01](https://user-images.githubusercontent.com/7655454/204023880-c141d341-fef1-49ef-9653-158f04ee66ea.png)

Reply:
I have now changed this to practical 1 and 2, and renumbered the rest